amqp1jobs/config.go

```
@@ -0,0 +1,197 @@
+	deleteOnStop  string = "delete_queue_on_stop"
+	priority      string = "priority"
+	multipleAck   string = "multiple_ack"
+	requeueOnFail string = "requeue_on_fail"
+
+	// new in 2.12
+	redialTimeout      string = "redial_timeout"
+	exchangeDurable    string = "exchange_durable"
+	exchangeAutoDelete string = "exchange_auto_delete"
+	queueAutoDelete    string = "queue_auto_delete"
+
+	dlx           string = "x-dead-letter-exchange"
```

golangci-lint (1.64.8)

const `dlx` is unused

(unused)